server/configuration.go

```diff
@@ -40,6 +40,12 @@ func (c *configuration) ProcessConfiguration() {
 	c.ClientSecret = strings.TrimSpace(c.ClientSecret)
 	c.EncryptionKey = strings.TrimSpace(c.EncryptionKey)
 	c.WebhookSecret = strings.TrimSpace(c.WebhookSecret)
+	if c.MaxSizeForCompleteDownload < 0 {
```

Comment:
There's a bug with the system console where values can sometimes not get saved, making it brutal to keep losing the various secrets. If this happens for these values, just use sensible values.

Reply:
So what are we actually fixing here, some default values that were missing? Anyone looking at the bug you mention?

Reply:
I'm trying to work around https://community.mattermost.com/core/pl/o78pcxfj87f4frt3qhrqxw1x1y, specifically.